recipes/nd2/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "nd2" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3f7e9c6c8c73cd40ba908e46e7f5ba49794c0755e3b7677e0e48c5e4b011e07e
+
+build:
+  number: 0
+  skip: True  # [win or py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
+    - libgomp  # [linux]
+  host:
+    - python
+    - pip
+    - cython
+    - numpy
+    - libtiff
+    - libzlib
+    - jpeg
+    - setuptools_scm
+  run:
+    - python
+    - lxml
+    - {{ pin_compatible('numpy') }}
+    - typing_extensions
+    - imagecodecs
+    - wurlitzer  # [not win]
+
+test:
+  imports:
+    - nd2
+    - nd2._sdk.latest
+    - nd2._legacy
+    - imagecodecs._jpeg2k
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/tlambert03/nd2
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

We need to include the sdk license here too. This field can be a list of paths so, something like this:

```suggestion
  license_file: 
    - LICENSE
    - src/sdk/LICENSE
```

I don't know if the `license` and `license_family` fields need to be updated to reflect this too.